test/test_helper.rb

```diff
@@ -25,4 +25,8 @@ def authenticate_with_github(user)
           }
         })
   end
+
+  def return_gems
+    gems = [{'name'=>'gem1', 'info'=>'This is Gem1.'}, {'name'=>'gem2', 'info'=>'This is Gem2.'}, {'name'=>'gem3', 'info'=>'This is Gem3.'}]
+  end
```

Comment:
Good initiative with this, but since we will be relying on responses form RubyGems.org API we should probably decide on a strategy for our test, I suggest you look at the [`vcr` gem](https://github.com/vcr/vcr) which records HTTP requests  and replays them when running the test suite.